plaidml/bridge/keras/backend_test.py

```diff
@@ -589,6 +587,7 @@ def testDivConstant(self, b, x, c):
             c / x,
         ]
 
+    @unittest.skip("Failing on llvm.cpu")
```

Comment:
Does this crash or just fail? If it fails, why do we skip it?

Reply:
Didn't know that we skip only hard crashing tests. Fixed.